chains/berabartio/metadata.yaml

```diff
@@ -23,6 +23,4 @@ nativeToken:
   symbol: BERA
 protocol: ethereum
 rpcUrls:
-  - http: https://bartio.rpc.berachain.com/
-  - http: https://bartio.drpc.org
-  - http: https://bera-testnet.nodeinfra.com
+  - http: https://berachain-testnet-rpc.publicnode.com
```

Comment:
this is wrong i think - because the bera bartio testnet has been deprecated?

Reply:
yeah the chainids are different
```
cast chain-id --rpc-url https://berachain-testnet-rpc.publicnode.com
80069
```

Reply:
do we have a test to ensure that every rpc added matches the chainid in the metadata?

Reply:
damn i fell into the trap, many testnets seem to have been deprecated (probably a job for another pr)

Reply:
doesn't seems so otherwise it would have complained, seems a good idea to add it here now

Reply:
added the chain id check in the test